testbench/grpc_server.py

```diff
@@ -43,6 +43,34 @@ def CreateBucket(self, request, context):
         self.db.insert_bucket(request, bucket, context)
         return bucket.metadata
 
+    def UpdateBucket(self, request, context):
+        self.db.insert_test_bucket()
+        intersection = field_mask_pb2.FieldMask(
+            paths=[
+                "name",
+                "bucket_id",
+                "project",
+                "metageneration",
+                "location",
+                "location_type",
+                "create_time",
+                "update_time",
+                "owner",
+            ]
+        )
+        intersection.Intersect(intersection, request.update_mask)
+        if len(intersection.paths) != 0:
+            return testbench.error.invalid(
+                "UpdateObject() cannot modify immutable Object fields [%s]"
```

Comment:
Error message should say Bucket rather than Object.

Reply:
Ooops, fixed.